File: app/src/main/java/fr/insapp/insapp/models/EventSearchResults.java
package fr.insapp.insapp.models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by thomas on 15/07/2017.
 */

public class EventSearchResults {

    @SerializedName("events")
    private List<Event> events;

    public EventSearchResults(List<Event> events) {
        this.events = events;
    }

    public List<Event> getEvents() {
        return events;
    }
}

File: app/src/main/java/fr/insapp/insapp/models/EventSearchResults.kt
package fr.insapp.insapp.models

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

/**
 * Created by thomas on 15/07/2017.
 * Kotlin rewrite on 30/08/2019.
 */

@Parcelize
data class EventSearchResults(
    @SerializedName("events") val events: MutableList<Event>
): Parcelable
